package.json

```diff
@@ -5,18 +5,19 @@
   "main": "index.js",
   "scripts": {
     "coverage": "codecov",
-    "test": "standard && nyc --reporter=lcov mocha"
+    "lint": "standard",
+    "test": "nyc --reporter=lcov mocha"
```

Comment:
Why separating the scripts? I don't see any benefit, but the drawback that no linting is done before making a commit. Also, it generates more boilerplate in the CI config.

Reply:
When I run the tests I don't want standard to be in the way. I'm writing code and writing tests, I want to run the tests. With a proper linting config, autofixing on save would be easy, but here there's no linting config so that's part of the issue.

> no linting is done before making a commit

Nothing changed on this front